packages/fast-tooling/.eslintrc.cjs

```diff
@@ -3,5 +3,23 @@ module.exports = {
     rules: {
         "@typescript-eslint/class-name-casing": "off",
         "@typescript-eslint/explicit-function-return-type": "off",
+        "import/extensions": ["error", "always", {
+            "props": "never",
+            "mdn-data": "never",
+            "definition": "never",
+            "utilities": "never",
+            "service": "never",
```

Comment:
@janechu if there are any "extensions" you noticed that I've missed let me know. We have to opt extensions *out* unfortunatly, and this package has a lot of filenames like `"foo.utilities.ts"` and `"bar.service.ts"`